less/elements/pieces/global.less

```diff
@@ -31,22 +30,14 @@ h1, h2, h3, h4, h5, h6,
 	margin-bottom: .5em;
 }
 
-h1, .h1{ font-size: ~'clamp(@{font-size-h1-m}, 4vw + 1rem, @{font-size-h1})'; }
-h2, .h2{ font-size: ~'clamp(@{font-size-h2-m}, 3.3vw + 1rem, @{font-size-h2})'; }
-h3, .h3{ font-size: ~'clamp(@{font-size-h3-m}, 2.7vw + 1rem, @{font-size-h3})';  }
-h4, .h4{ font-size: ~'clamp(@{font-size-h4-m}, 2.2vw + 1rem, @{font-size-h4})';  }
-h5, .h5{ font-size: ~'clamp(@{font-size-h5-m}, 1.8vw + 1rem, @{font-size-h5})'; }
-h6, .h6{ font-size: ~'clamp(@{font-size-h6-m}, 1.8vw + 1rem, @{font-size-h6})'; }
-
-/* 
-    May want something like this for very large headlines that might 
-    overflow the screen can also be used on links showing long urls 
-*/
-// h1,h2,h3,h4,
-// .h1,.h2,.h3,.h4{
-// 	overflow-wrap: break-word; // Help prevent overflow of long words/names/URLs
-// 	hyphens: auto; // Optional, not supported for all languages
-// }
+//https://min-max-calculator.9elements.com/
+//max fontsize value at 800px, min fontsize value at 400px
+h1, .h1{ font-size: ~'clamp(@{font-size-h1-m}, 2.25rem + 3vw, @{font-size-h1})'; }
+h2, .h2{ font-size: ~'clamp(@{font-size-h2-m}, 0.625rem + 5rem, @{font-size-h2})'; }
```

Comment:
For the h2 calculation, is there supposed to be a 'vw' instead of both numbers being 'rem'?

Reply:
yep, you are correct its supposed 5rem should be 5vw. Its fixed now